controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/RunSerializer.java

```diff
@@ -377,7 +377,8 @@ static String valueOf(RunStatus status) {
     static RunStatus runStatusOf(String status) {
         switch (status) {
             case "running"                    : return running;
-            case "outOfCapacity"              : return outOfCapacity;
+            case "outOfCapacity"              : return nodeAllocationFailure;  // TODO: Remove after March 2022
```

Comment:
We don't re-write historic runs, so this needs to stay, unless we start re-writing all of those regularly as well.

Reply:
When do they expire?

Reply:
When they're more than 65 runs old, or somehting, and they're also no longer the "first failing" or "lasts successful" run. Potentially never. I guess we should re-write these runs ...

Reply:
I can do that in a differnet PR.

Reply:
Never mind I'm just bad at reading code. We update these as well.

Reply:
Thanks. I'll remove the date for now.